packages/web/config/generate-chain-infos/source-chain-infos.ts

```diff
@@ -3990,6 +3998,34 @@ const mainnetChainInfos: SimplifiedChainInfo[] = [
     features: ["ibc-transfer", "ibc-go"],
     explorerUrlToTx: "https://www.mintscan.io/sei/txs/{txHash}",
   },
+  {
+    rpc: "https://fx-json.functionx.io",
+    rest: "https://fx-rest.functionx.io",
+    chainId: "fxcore",
+    chainName: "f(x)Core",
+    bip44: {
+      coinType: 60,
+    },
+    bech32Config: Bech32Address.defaultBech32Config("fx"),
+    currencies: [
+      {
+        coinDenom: "FX",
+        coinMinimalDenom: "FX",
+        coinDecimals: 18,
```

Comment:
As per the Chain Registry, the decimals should be 0--meaning FX is indivisible.
https://github.com/cosmos/chain-registry/blob/master/fxcore/assetlist.json

Reply:
resolved